exercises/bracket-push/example.scala

```diff
@@ -0,0 +1,23 @@
+import scala.util.parsing.combinator.RegexParsers
+
+object Brackets extends RegexParsers {
```

Comment:
interesting. :-)
I have to admit that RegexParsers would not have come to my mind as a first idea, but simply keeping track of the various brackets by just counting them. your solution is elegant but may be too advanced for some? (or at least mention to use RegexParsers explicitly in the Readme then?).
or perhaps have exercise sgf-parsing instead for this kind of solution?